src/lib/components/file-browser/file-list/file-list.component.ts

```diff
@@ -490,6 +498,7 @@ export class TerraFileListComponent implements OnInit, AfterViewInit, OnChanges,
                     icon: 'icon-delete',
                     clickFunction: (event: Event): void => {
                         this._objectsToDelete = [storageObject];
+                        this._openDeleteDialog();
```

Comment:
Funktioniert das denn so? Denn _objectsToDelete wird in dieser Methode ja wieder überschrieben

Reply:
Ich denke es macht Sinn das aus der _openDeleteDialog Methode rauszunehmen

Reply:
Ist geändert und funktioniert